### PR TITLE
Bump copier to 7.0.1

### PR DESCRIPTION
**Pull request by fcollonval**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by fcollonval** [comment]
@conda-forge-admin, please rerender

**Comment by fcollonval** [comment]
@kir0ul this superseeds #2 if you want I can help maintaining the recipe.

File: .scripts/build_steps.sh [hunk 1]
```diff
@@ -33,9 +33,9 @@ CONDARC
 
 
 mamba install --update-specs --yes --quiet --channel conda-forge \
-    conda-build pip boa conda-forge-ci-setup=3
+    conda-build pip boa conda-forge-ci-setup=3 "py-lief<0.12"
 mamba update --update-specs --yes --quiet --channel conda-forge \
-    conda-build pip boa conda-forge-ci-setup=3
+    conda-build pip boa conda-forge-ci-setup=3 "py-lief<0.12"
```
Comment (kir0ul) [inline .scripts/build_steps.sh:38]: May I ask what's the purpose of adding `py-lief`?

**Comment by kir0ul** [comment]
> @kir0ul this superseeds #2 if you want I can help maintaining the recipe.

Hey @fcollonval, thanks for the PR! Sure, that'd be great if you can help maintain the recipe! Could you add yourself to the maintainers list?

File: .scripts/build_steps.sh (hunk 1 above)
Comment (fcollonval) [reply .scripts/build_steps.sh:38]: No idea, this was added by the bot 😉

**Comment by fcollonval** [comment]
Thanks @kir0ul 
I added myself as maintainer

**Pull request merged**